.circleci/config.yml

```diff
@@ -255,8 +255,8 @@ jobs:
           working_directory: ~/expo
```

Comment:
revert this change. if the cherry-pick pulls in changes that don't make sense on the sdk-xx branch, manually exclude those from the commits. for example, this file references bare-expo which doesn't make sense on the sdk-34 branch.